pkg/skaffold/build/parallel.go

```diff
@@ -34,6 +34,10 @@ type artifactBuilder func(ctx context.Context, out io.Writer, tagger tag.Tagger,
 
 // InParallel builds a list of artifacts in parallel but prints the logs in sequential order.
 func InParallel(ctx context.Context, out io.Writer, tagger tag.Tagger, artifacts []*latest.Artifact, buildArtifact artifactBuilder) ([]Artifact, error) {
+	if len(artifacts) == 1 {
```

Comment:
Why can't InParallel handle the single case?

Reply:
inParallel can handle the single case but it's extra work on streams. More important than that, it loses a bit the colors. I couldn't fix it. 

Building in // should work since the secret is shared by all jobs. It's defined at the builder level not the artifact level.

Reply:
SGTM